dev-libs/libcgroup/libcgroup-0.41-r6.ebuild

```diff
@@ -0,0 +1,107 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools flag-o-matic linux-info pam systemd
+
+DESCRIPTION="Tools and libraries to configure and manage kernel control groups"
+HOMEPAGE="http://libcg.sourceforge.net/"
+SRC_URI="mirror://sourceforge/project/libcg/${PN}/v${PV}/${P}.tar.bz2"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
+
+IUSE="+daemon elibc_musl pam static-libs test +tools"
+REQUIRED_USE="daemon? ( tools )"
+
+BDEPEND="
+	sys-devel/bison
+	sys-devel/flex
+	elibc_musl? ( sys-libs/fts-standalone )
+"
+DEPEND="pam? ( virtual/pam )"
+RDEPEND="${DEPEND}"
+
+DOCS=(README_daemon README README_systemd INSTALL)
```

Comment:
is this definition necessary? try removing it and seeing whether `default`/`einstalldocs` catches them all

Reply:
Done